### ISSUE TITLE
Ng-click doesn't works when node is draggable?

### ISSUE THREAD

**Issue by amurtola** [angular-ui-tree/angular-ui-tree#494]
I whant to select node onclick, but ng-click starts to work only after adding "data-nodrag" to the node content, but drag and drop is not working after this. How to anable ng-click with drag and drop?

**Comment by guisouza** [comment]
u'll have to create a switch that toggle the data-nodrag, so the user will have to turn it before sorting the items .

**Comment by maeishoj** [comment]
can you elaborate more on it?

I want to "highlight" a node on click and set node.selected property to true when clicked.
I am not able to do that.. any help would be appreciated..

**Comment by sunlei33** [comment]
try to set data-drag-delay(such as 500), I fixed by this

**Comment by blackat** [comment]
Hi,
I have solved using the attribute `data-nodrag` in this way

`<a ng-click="showDetails(this)" data-nodrag>{{node.title}}</a>`

I hope it helps.

**Comment by jacobscarter** [comment]
Closing this issue due to inactivity. See #815 for more information.
Please create a new issue is this still occurs with the current version of the component.
